pennylane/tape/tapes/qubit_param_shift.py

```diff
@@ -129,6 +129,9 @@ def parameter_shift(self, idx, params, **options):
             idx (int): trainable parameter index to differentiate with respect to
             params (list[Any]): the quantum tape operation parameters
 
+        Keyword Args:

```

Comment:
Ah so the qubit tape updates are here! (https://github.com/PennyLaneAI/pennylane/pull/909#discussion_r529014797)

Reply:
Yep, this is because as I was adding these, I realised it was the only gradient method missing the changes 😆 I'll backport it to #909

Reply:
done